sdk/healthinsights/azure-healthinsights-radiologyinsights/samples/sample_age_mismatch_inference_async.py

```diff
@@ -0,0 +1,171 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+import asyncio
+import datetime
+import os
+import uuid
+
+
+from azure.core.credentials import AzureKeyCredential
+from azure.healthinsights.radiologyinsights.aio import RadiologyInsightsClient
+from azure.healthinsights.radiologyinsights import models
+
+"""
+FILE: sample_age_mismatch_inference_async.py
+
+DESCRIPTION:
+The sample_age_mismatch_inference_async.py module processes a sample radiology document with the Radiology Insights service.
+It will initialize an asynchronous RadiologyInsightsClient, build a Radiology Insights request with the sample document,
+submit it to the client, RadiologyInsightsClient, and display 
+-the Age Mismatch patient ID,
+-the Age Mismatch url extension,
+-the Age Mismatch offset extension,
+-the Age Mismatch length extension, and
+-the Age Mismatch evidence
+
+
+USAGE:
+
+1. Set the environment variables with your own values before running the sample:
+    - AZURE_HEALTH_INSIGHTS_API_KEY - your source from Health Insights API key.
+    - AZURE_HEALTH_INSIGHTS_ENDPOINT - the endpoint to your source Health Insights resource.
+
+2. python sample_age_mismatch_inference_async.py
+"""
+
+
+class HealthInsightsSamples:
```

Comment:
Let's remove the class definition and just leave the function definition. Here is an example you could take a look: https://github.com/Azure/azure-sdk-for-python/blob/81a8cdaea391ee6132bbd422f885179d15d39765/sdk/formrecognizer/azure-ai-formrecognizer/samples/v3.2_and_later/sample_analyze_business_cards.py